vnext/Microsoft.ReactNative.Cxx/StructInfo.h

```diff
@@ -21,14 +21,18 @@
 // Please skip below to read about REACT_STRUCT and REACT_FIELD macros.
 //
 
-#define INTERNAL_REACT_STRUCT(structType)                                                  \
+#define INTERNAL_REACT_STRUCT_GETSTRUCTINFO(structType)                                    \
```

Comment:
>INTERNAL_REACT_STRUCT_GETSTRUCTINFO [](http://example.com/codeflow?start=8&length=35)

Let's discuss it offline: I would prefer if we do not introduce a new macro here because it has not difference with the exisiting INTERNAL_REACT_STRUCT macro.

Reply:
Actually it removes the forward declaration, because a forward declaration is not able to declare something in another class.